### PR TITLE
Changes needed to build and run on iOS Device

### PR DESCRIPTION
**Pull request by thomascannon**
Wanted to get this base for iOS compatibility in Main. From there I/others can work on the bugs to make it work better on iOS.

Note that I'm building and signing this with my own Developer account and so the BundleID was set to net.thomascannon.chameleonultragui. This won't stop anyone else building and running it, but we should obviously update it to the project's BundleID and sign with the team's Dev account before releasing the app. Let me know when we get that dev account sorted and you can add me.

### CODE REVIEW

File: chameleonultragui/ios/Podfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
+platform :ios, '15.0'
```
Comment (Foxushka) [inline chameleonultragui/ios/Podfile:1]: Does this mean that we have minimal iOS version 15? Or it is just target version?

**Comment by thomascannon** [comment]
Also to add, if you see the changes/additions for OSX, like the Podfile, this wasn't a mistake. Whenever I build for iOS it creates those file for OSX as well as iOS. I don't know why TBH, but I left them in the commit because they seem to be needed.

File: chameleonultragui/ios/Podfile (hunk 1 above)
Comment (thomascannon) [reply chameleonultragui/ios/Podfile:1]: This was the default. If I go older XCode complains when it builds, maybe some libraries are not compatible, I didn't look in detail. We are on iOS 16+ now which supports the majority of devices in use.

**Review by Foxushka — APPROVED** [review]
Thank you for your work

**Pull request merged**